docs/guide/cache/symfony_reverse_proxy.md

```diff
@@ -38,16 +38,17 @@ The configuration of [[= product_name =]] for using Varnish or Fastly requires a
 Before you configure Symfony to [work behind a load balancer or a reverse proxy](https://symfony.com/doc/5.1/deployment/proxies.html),
 make sure that [Symfony reverse proxy](#symfony-reverse-proxy) is enabled.
 
-Set the following environment variable:
-- `TRUSTED_PROXIES`: String with trusted IP, multiple proxies can be configured with a comma, for example, `TRUSTED_PROXIES="192.0.0.1,10.0.0.0/8"`
+To set environment variables for `TRUSTED_PROXIES` and `TRUSTED_HOSTS`, use [Symfony semantic configuration](https://symfony.com/doc/current/deployment/proxies.html#solution-settrustedproxies).
```

Comment:
I think there's a bit of misunderstanding here. This does not set any environment variables. Previously it was possible to *configure* trusted proxies via `TRUSTED_PROXIES` env varibale. Now to configure trusted proxies you need to\* use Symfony semantic configuration under `framework.trusted_proxies` root to do so.


\* technically it's possible to still use our own `TRUSTED_PROXIES` variable, but this is not aligned with the direction Symfony went.